stdlib/public/Reflection/MetadataSource.cpp

```diff
@@ -19,70 +19,69 @@ using namespace reflection;
 
 class PrintMetadataSource
   : public MetadataSourceVisitor<PrintMetadataSource, void> {
-  std::ostream &OS;
+  FILE *file;
   unsigned Indent;
 
-  std::ostream &indent(unsigned Amount) {
+  FILE * &indent(unsigned Amount) {
```

Comment:
These should all drop the `&`. (They probably shouldn't return anything at this point since they're not going to be chained.)

Reply:
Done